src/lib.rs

```diff
@@ -1386,6 +1571,103 @@ mod tests {
         }
     }
 
+    fn expect_opaque_sequence(
+        input: &[u8],
+        kind: OpaqueSequenceKind,
+        expected_payload: &[u8],
+        expected_trailer: &[Sequence],
+    ) {
+        let mut expected_dispatched: Vec<Sequence> = vec![Sequence::OpaqueStart(kind)];
```

Comment:
```suggestion
        let mut expected_dispatched = vec![Sequence::OpaqueStart(kind)];
```

Is this really necessary? I'd be extremely surprised if Rust wasn't able to infer it here.